radicle-httpd/src/api/v1/delegates.rs

```diff
@@ -40,7 +40,8 @@ async fn delegates_projects_handler(
         .filter_map(|id| {
             let Ok(repo) = storage.repository(id) else { return None };
             let Ok((_, head)) = repo.head() else { return None };
-            let Ok(doc) = repo.identity_of(ctx.profile.id()) else { return None };
+            let Ok((_, doc)) = repo.identity_doc() else { return None };
+            let Ok(doc) = doc.verified() else { return None };
```

Comment:
This only covers the `/delegates/` path, probably we also want this in `/projects/`

Reply:
There were 2 other usages, [changed](https://github.com/radicle-dev/heartwood/compare/c21855952119900dde0be1616095aa5ef9d8ef8b..33dd5707e56a08766a9b6e7cbb90ef8da8e1b22e)